### PR TITLE
Fish completion

### PR DESCRIPTION
**Pull request by shigemk2**
refered: #656

### CODE REVIEW

**Comment by mislav** [comment]
Thanks for the contribution! I'll need some time to review this. Does this also work when `hub` is aliased as `git` in fish?

**Comment by shigemk2** [comment]
Sorry for bothering you.

> Does this also work when hub is aliased as git in fish?

But tab completion does not work when hub is aliased as git in fish. I have to check and rewrite fish shell code.

**Comment by shigemk2** [comment]
I made this PR work when hub is aliased as git in fish.
https://github.com/fish-shell/fish-shell/issues/3767

File: commands/alias.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func alias(command *Command, args *Args) {
 		var eval string
 		switch shell {
 		case "fish":
-			eval = `function git --description 'Alias for hub, which wraps git to provide extra functionality with GitHub.'
```
Comment (mislav) [inline commands/alias.go]: Is it possible to also preserve the `--description`?

Comment (shigemk2) [inline commands/alias.go]: Yes. I can preserve `--description` like this:

```sh
function git --wraps hub --description 'Alias for hub, which wraps git to provide extra functionality with GitHub.'
    hub $argv
end
```

Comment (shigemk2) [inline commands/alias.go]: I fixed.

**Comment by shigemk2** [comment]
I fixed PR.

**Comment by shigemk2** [comment]
Thank you for your merge!

**Pull request merged**